rules.yaml

```diff
@@ -1033,6 +1033,18 @@ rules:
         key: _Z_DATA
         value: ${XDG_CACHE_HOME}/z
 
+  - name: julia
+    dotfile:
+      name: .julia
+      is_dir: true
+    actions:
+      - type: migrate
+        source: ${HOME}/.julia
+        dest: ${XDG_DATA_HOME}/julia
+      - type: export
+        key: JULIA_DEPOT_PATH
+        value: ${XDG_DATA_HOME}/julia:${JULIA_DEPOT_PATH}
```

Comment:
@Grafcube why is it needed to prepend the path? Why not just `value: ${XDG_DATA_HOME}/julia`?

Reply:
It's copied from this [Arch wiki entry](https://wiki.archlinux.org/title/XDG_Base_Directory). As for the reason, it's explained on the Julia docs here: https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH